src/workerd/api/html-rewriter.h

```diff
@@ -241,6 +241,9 @@ class Element final: public HTMLRewriter::Token {
   kj::Maybe<Impl> impl;
 
   void htmlContentScopeEnd() override;
+
+  template <auto Func, auto StreamingFunc>
+  jsg::Ref<Element> contentRewriter(Content content, jsg::Optional<ContentOptions> options);
```

Comment:
Nit: I'd find `rewriteContent()` to be a bit clearer of a name, or maybe even `rewriteContentGeneric()` with a doc comment explaining it's the shared implementation of all of the `before()`/`after()`/etc. functions.